manifests/charts/gateways/istio-ingress/templates/preconfigured.yaml

```diff
@@ -47,10 +48,11 @@ spec:
       value:
         name: "envoy.filters.network.tcp_cluster_rewrite"
         config:
-          cluster_pattern: "\\.global$"
+          cluster_pattern: "\\.{{ .Values.global.multiCluster.globalDomainSuffix }}$"
```

Comment:
Same here, better to trim space around the parameter `global.multiCluster.globalDomainSuffix`.